vtysh/bgp_vty.c

```diff
@@ -4875,6 +4934,8 @@ cli_neighbor_timers_execute(char *vrf_name, int argc, const char *argv[])
     const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor;
```

Comment:
How did you solve the issue pointed in the ticket?
I mean: Original quagga does not allow to change configuration for a neighbor included into peer-group.